apps/site/lib/site_web/controllers/stop_controller.ex

```diff
@@ -214,6 +239,7 @@ defmodule SiteWeb.StopController do
     case Services.Repo.by_route_id(route_id) do
       [%Service{} | _] = services ->
         services
+        |> Enum.reject(&(&1.id === "canonical"))
```

Comment:
question: Could these be filtered upfront using `Services.Repo.by_route_id` params?

Reply:
it's a negation, so I don't think so?

Reply:
Ah, I was thinking of the `filter[canonical]=false` filter supported on the [/route_patterns](https://api-v3.mbta.com/docs/swagger/index.html#/RoutePattern) endpoint.

Reply:
Frankly it's too weird that there is even any service with an ID of "canonical", but I did see it 😬